src/Ilios/AuthenticationBundle/Service/PermissionChecker.php

```diff
@@ -45,6 +45,12 @@ class PermissionChecker
     const CAN_DELETE_SESSION_TYPES = 'canDeleteSessionTypes';
     /** @var string */
     const CAN_CREATE_SESSION_TYPES = 'canCreateSessionTypes';
+    /** @var string */
+    const CAN_READ_SCHOOLS = 'canReadSchools';
```

Comment:
This might be overly picky, but no one will have the ability to read plural schools. I feel like this should be CAN_READ_SCHOOL (singular) and the same for update and delete.